internal/orchestrator/poller.go

```diff
@@ -45,6 +46,12 @@ func NewPoller(rpc common.RPC, storage storage.IStorage) *Poller {
 		lastPolledBlock = new(big.Int).Sub(pollFromBlock, big.NewInt(1)) // needs to include the first block
 		log.Warn().Err(err).Msgf("No last polled block found, setting to %s", lastPolledBlock.String())
 	} else {
+		// if the configured from block is less than the last polled block,
```

Comment:
This is changing in the next stack

Reply:
good, because this is faulty behaviour. If the service crashes, then it can't choose the configured block